### ISSUE TITLE
[Dataset Title/Name]: Global surface and canopy urban heat island intensity dataset

### ISSUE THREAD

**Issue by TC25** [samapriya/awesome-gee-community-datasets#276]
### Contact Details

tc.chakraborty@pnnl.gov or qiquanyang@tongji.edu.cn

### Dataset description

A global dataset of urban heat island using multiple methods and including estimates for both air temperature and land surface temperature. It is available monthly from 2003 to 2020 (from 2001 for the dataset from MODIS Terra satellite).

Link to data: https://pan.baidu.com/s/1qm8MUG1Ny5uSVFqWsWbd3g 
Code：1234

The readme file in that directory contains full information about the data product.

Link to paper: https://authors.elsevier.com/c/1jWMM7qzT3EKU

### Earth Engine Snippet if dataset already in GEE

for example
```js
var asset_name = ee.ImageCollection("path to your collection")
```
Sample Code: Add a sample code maybe just adding your datasets in the code editor

### Enter license information

CC-BY-4.0

### Keywords

urban, heat, climate, city

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samapriya** [comment]
Can you provide more details what is the total size

**Comment by TC25** [comment]
Around 160 GB.

**Comment by rodrigo-j-goncalves** [comment]
Hi, the "Link to data" is broken.

Also, when citing a published article, dataset, chapter, etc. I suggest using the DOI instead of a link to the publisher. 
In this case, it should be [this one](https://doi.org/10.1016/j.rse.2024.114343).

**Comment by TC25** [comment]
That data link works on my personal computer (but not my work computer). Make sure you are not on any VPN that blocks Chinese sites. 

Once you reach the page, you should be able to enter the password to enter the directory.

**Comment by samapriya** [comment]
Hey @TC25 
I can get to the site I can browse but I cannot download the files without a Baidu account which I am not interested in creating let me know if there is an alternative at this point your Google Drive solution might work better than this

**Comment by TC25** [comment]
The data have been uploaded to Google Drive: https://drive.google.com/drive/u/5/folders/1dKwW2ceg457iM2UHLJWmMcNhkx5OfBY9

Please list Yifan Cheng (yifanc17@illinois.edu) as a data curator as she moved all the files from Baidu to Drive. Thanks!

**Comment by samapriya** [comment]
Okay I have all these ingested but I need to know more details for the data description. @TC25 Can you provide me with a table describing each dataset maybe in a table. Here are the collections paths they are not public yet

```
projects/sat-io/open-datasets/UHII/AMOD2
projects/sat-io/open-datasets/UHII/MOD1
projects/sat-io/open-datasets/UHII/MOD2
projects/sat-io/open-datasets/UHII/MYD1
projects/sat-io/open-datasets/UHII/MYD2
projects/sat-io/open-datasets/UHII/SAT
projects/sat-io/open-datasets/UHII/SMOD2
projects/sat-io/open-datasets/UHII/SMYD1
```

**Comment by TC25** [comment]
Let me know if you need any more details: https://docs.google.com/document/d/182ANRafrLySyiVKgXu7rvGdsBX1DLHArXDoOiTEJZvM/edit

**Comment by samapriya** [comment]
thanks this should do

**Comment by samapriya** [comment]
This has now been completed , thank you for contributing. Check changelog for links to the datasets.

**Comment by samapriya** [comment]
Hey @TC25 
I know this ticket is closed but I am trying to actually understand how the date format works 

I see dates like so my guess was June 2013

```
projects/sat-io/open-datasets/UHII/AMOD2/AMod2_Nig_2013_6
```

but then I also see and I don't know what to guess 2013_24 what does it mean here

```
projects/sat-io/open-datasets/UHII/AMOD2/AMod2_Nig_2014_24
```

**Comment by TC25** [comment]
Thanks, Sam!

Here is a text you can add to the data descriptor. Let me know if it makes sense.

The image names use the format “Indicator_Diurnal_Year_Month”. The “Indicator” field represents temperature data corresponding to the UHII, where, for instance, “Mod1” denotes UHII derived from MOD11A1 data. The “Diurnal” field can be either “Day” or “Nig”, signifying daytime and nighttime UHII, respectively. "Year” denotes the UHII year, and "Month” indicates the specific month. 

Besides monthly UHII results, we also provide quarterly and annual averages. When “Month” part of the image name takes values from 1 to 12, it signifies the monthly average (from January to December). If “Month” is between 21 and 24, it indicates the quarterly average (21 for March-May, 22 for June-August, 23 for September-November, and 24 for December-February). When “Month” is 30, it denotes the annual average.

**Comment by samapriya** [comment]
Reopening this issue seems some of the files were not downloaded from Google Drive reprocessing the collections in place will keep you all updated

**Comment by samapriya** [comment]
Hey @TC25 
Can you do me a favor and give me the count per indicator folder for example I found
MOD2 has 452 files
MOD1 has 382 files

Do those numbers seem correct to you and can you create this for all of the indicator folders

**Comment by samapriya** [comment]
Okay all collections have been updated. Based on the information you provided new metadata has been created such as cadence, diurnal and so on here is an example. It now also has start and end dates based on the additional information you provided. I will updated the file to include that as a note including the scaling factor to get to Celcius. Closing this ticket again

```
cadence: monthly
diurnal: daytime
id_no: Mod1_Day_2001_11
system:index: Mod1_Day_2001_11
system:time_end: 1007078400000
system:time_start: 1004572800000
year: 2001
```

**Comment by TC25** [comment]
Thanks, Sam! Do you still need me to check the number of files?

**Comment by samapriya** [comment]
Yeah feel free to check the collection size and match it to what you would expect but I pushed the updated and properties last night and added your note

**Comment by leeyuchen2022** [comment]
Hello, why is your data not yearly or monthly? Some data is missing. What is the reason? I am waiting for your reply. This is very important to me.

**Comment by leeyuchen2022** [comment]
For example, the 22 series of AMod2 only covers up to 2019, so the data is not very complete. 
![Uploading 1732695553108.png…]()

**Comment by TC25** [comment]
@Samapriya Roy ***@***.***> When I look in the original Google
Drive, I can see the AMod2 data up to 2020:
https://drive.google.com/drive/folders/1WQ_NFvgsYuopdkRkOquWtofjBlbdA8J4

Can you check what might have happened?

**Comment by samapriya** [comment]
I will take a look but it may not happen for abit with AGU travels and more. In any case I am reopening the issue so it's on my radar

**Comment by TC25** [comment]
No problem. Another I noticed is that 'projects/sat-io/open-datasets/UHII/SAT/SAT_Nig_2015_22' is on Earth Engine, but available on the Google Drive: https://drive.google.com/drive/u/1/folders/1ZvVTXVXWsPadnmBnr8lL11xD37NOIgH1

**Comment by TC25** [comment]
There is now a new version of the Global Urban Heat Island Intensity Dataset available.

(1) The base map for urban area delineation has been updated from MODIS land cover to ESA CCI land cover.

(2) The dataset now provides the spatial extents of both urban and rural areas (background reference areas).

(3) In addition to urban heat island intensity, the dataset now also includes the average urban temperature.

You can download the updated dataset from the following link:
https://figshare.com/articles/dataset/Global_Urban_Heat_Island_Intensity_Dataset_Version2_/30102847
